src/core/Akka/IO/ByteBuffer.cs

```diff
@@ -0,0 +1,115 @@
+//-----------------------------------------------------------------------
+// <copyright file="ActorBase.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+
+namespace Akka.IO
+{
+    public class ByteBuffer
```

Comment:
This is pretty similar, although Helios' `ByteBuffer` offers more methods for doing things like "just add an integer to my byte array" and takes care of the underlying bit conversion.

I think using this class is probably fine. Anyone who's going to be doing socket IO with Akka.NET is already going to be familiar with byte-formatting their messages.